.github/workflows/web.yml

```diff
@@ -683,3 +683,14 @@ jobs:
           major_version: ${{ needs.get-version.outputs.major_version }}
           minor_version: ${{ needs.get-version.outputs.minor_version }}
           stability: ${{ needs.get-version.outputs.stability }}
+
+  deploy-platform-nightly:
+    needs: [deliver-rpm]
+    if: github.event.schedule == '0 3 * * *'
```

Comment:
Use "${{ github.event_name }}" = "schedule"  to stay consistent with the get-version condition